main.css

```diff
@@ -153,7 +160,7 @@ h1, h2, h3 {
 
 .category-container {
     display: grid;
-    grid-template-columns: 12em 4em;
+    grid-template-columns: 14em 4em;
```

Comment:
Why is this being widened?

Reply:
filter are long, it looked better if there are less linebreaks as such I wide it a bit